app/femr/business/services/system/EncounterService.java

```diff
@@ -535,4 +535,38 @@ private Integer getTabFieldIdByTabFieldName(List<? extends ITabField> tabFields,
 
         return tabFieldId;
     }
+
```

Comment:
While not technically needed, it's good practice to add @Override annotation above this method to inform the compiler that you are overriding the method from IEncounterService. Same with the {@inheritdoc}. I noticed there's other methods that are missing this as well so i will edit before merging.